src/ui/public/doc_table/doc_table.html

```diff
@@ -20,7 +40,27 @@
           class="discover-table-row"></tr>
       </tbody>
     </table>
-  </paginate>
+    <!-- ToolBarFooter -->
+    <div class="kuilBar">
```

Comment:
And this can be updated too:

```html
    <div class="kuilBar docTableBar">
      <div class="kuiBarSection">
        <div
          ng-if="shouldShowLimitedResultsWarning()"
          class="kuiToolBarText kuiSubduedText">
          {{ limitedResultsWarning }}
        </div>
```